cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_service_based_app/templates.xml

```diff
@@ -17,7 +17,6 @@
     <increment name="devon4ng-app" description="CRUD devon4ng Service Based Angular">
       <incrementRef ref="app_angular_env"/>
       <incrementRef ref="app_angular_devon4j_component"/>
-      <incrementRef ref="crud_openapi_angular_client_app::app_angular_service_based"/>
     </increment>
```

Comment:
Please revert the changes from this file. Most probably you are using an old version.